packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js

```diff
@@ -163,6 +164,13 @@ class DemoWidget extends Component {
     return this.setState({initialActivity: value});
   }
 
+  @autobind
+  handleToggleStickyMode() {
```

Comment:
@mattnorris since we have access to state here, we are literally toggling the mode boolean. I can change it out if you'd like.